pandas/core/internals.py

```diff
@@ -4834,14 +4863,11 @@ def form_blocks(arrays, names, axes):
     if len(items_dict['ExtensionBlock']):
 
         external_blocks = []
+
         for i, _, array in items_dict['ExtensionBlock']:
-            if isinstance(array, ABCSeries):
-                array = array.values
-            # Allow our internal arrays to chose their block type.
-            block_type = getattr(array, '_block_type', ExtensionBlock)
             external_blocks.append(
-                make_block(array, klass=block_type,
-                           fastpath=True, placement=[i]))
+                make_block(array, klass=ExtensionBlock,
+                           placement=[i]))
```

Comment:
now this for loop is simplified, maybe use similar list comprehension pattern as above?